generator/generator_config.textproto

```diff
@@ -2563,6 +2563,26 @@ service {
 }
 
 # Spanner
+service {
+  service_proto_path: "google/spanner/v1/spanner.proto"
+  product_path: "google/cloud/spanner"
```

Comment:
Is there any reason to consider putting the generated files in a version-specific directory?

Reply:
I briefly considered it. I decided it doesn't matter because only internal files are generated in this configuration.